src/app/auth/auth-oidc/oidc-home/oidc-home.component.ts

```diff
@@ -27,9 +26,14 @@ export class LoginComponent implements OnInit {
   ) { }
 
   ngOnInit() {
-    this.configuration = this.oidcSecurityService.getConfiguration();
+    this.configuration = environment.oidcConfig;
     this.userData$ = this.oidcSecurityService.userData$;
-    this.isAuthenticated$ = this.oidcSecurityService.isAuthenticated$.pipe(pluck('isAuthenticated'));
+    this.isAuthenticated$ = this.oidcSecurityService.isAuthenticated$
+      .pipe(pluck('isAuthenticated'));
+
+    this.oidcSecurityService.isAuthenticated$.subscribe({
+      next: b => console.log('is authenticated? ', b),
+    });
```

Comment:
Shouldn't this be using the plucked version?
```suggestion
    this.isAuthenticated$.subscribe({
      next: b => console.log('is authenticated? ', b),
    });
```

Reply:
No, it's testing stuff. It shouldn't be there at all :)